### PR TITLE
add logistic regression to cloud bms

### PR DESCRIPTION
**Pull request by eordentlich**

### CODE REVIEW

**Comment by eordentlich** [comment]
changes do not impact pre-merge ci so can skip

File: python/run_benchmark.sh [hunk 1]
```diff
@@ -362,10 +362,10 @@ fi
 # Logistic Regression Classification
 if [[ "${MODE}" =~ "logistic_regression" ]] || [[ "${MODE}" == "all" ]]; then
     if [[ ! -d ${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32.parquet ]]; then
+    # same data set as random forest classifier
```
Comment (leewyang) [inline python/run_benchmark.sh:365]: assuming this is intentional?

**Review by leewyang — APPROVED** [review]

File: python/run_benchmark.sh (hunk 1 above)
Comment (lijinf2) [reply python/run_benchmark.sh:365]: I think so. LR benchmark can reuse the classification dataset of random forest classifier.

**Review by lijinf2 — APPROVED** [review]
Looks good!

**Pull request merged**